api/v1alpha1/skipobj_interfaces.go

```diff
@@ -21,3 +22,25 @@ type CommonSpec struct {
 	AccessPolicy *podtypes.AccessPolicy
 	GCP          *podtypes.GCP
 }
+
+func getVersionLabel(imageVersionString string) string {
```

Comment:
I would like a unit test of this: no tag, latest tag and hash

Reply:
Can do

Reply:
we should also update the integration tests, minimal for all operators at least.